aws-android-sdk-appsync/build.gradle

```diff
@@ -30,15 +30,17 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
-    compile 'com.google.code.findbugs:jsr305:3.0.1' // api
-    compile "com.squareup.okhttp3:okhttp:3.8.1" // impl
+    implementation 'com.google.code.findbugs:jsr305:3.0.1' // api
+    implementation "com.squareup.okhttp3:okhttp:3.8.1" // impl
 
     compile "com.amazonaws:aws-android-sdk-core:$aws_version" // api
     compileOnly "com.amazonaws:aws-android-sdk-cognitoidentityprovider:$aws_version"
     compileOnly "com.amazonaws:aws-android-sdk-s3:$aws_version"
 
-    compileOnly 'org.eclipse.paho:org.eclipse.paho.client.mqttv3:1.1.0'
```

Comment:
Should stay as compileOnly. This means that it won't be brought in as a dependency automatically.